sdk/PrebidMobile/PBBidManager.m

```diff
@@ -116,6 +116,7 @@ - (void)assertAdUnitRegistered:(NSString *)identifier {
 - (nullable NSDictionary<NSString *, NSString *> *)keywordsForWinningBidForAdUnit:(nonnull PBAdUnit *)adUnit {
     NSArray *bids = [self getBids:adUnit];
     [self startNewAuction:adUnit];
+    [self requestBidsForAdUnits:@[adUnit]];
```

Comment:
Is this the only place that startNewAuction being used?

Reply:
It's being used 2 times--here and in checkForBidsExpired.

Reply:
It's probably clearer to change the name too, to something like resetAdUnit to suit for the actual function.